internal/querynodev2/delegator/delegator_data.go

```diff
@@ -658,7 +668,7 @@ func (sd *shardDelegator) loadStreamDelete(ctx context.Context,
 				SegmentId:    info.GetSegmentID(),
 				PrimaryKeys:  storage.ParsePrimaryKeys2IDs(deleteData.Pks),
 				Timestamps:   deleteData.Tss,
-				Scope:        querypb.DataScope_Historical, // only sealed segment need to loadStreamDelete
```

Comment:
Why before L0 can we loadStreamDelete to Historical only? imo streamDelete should've been apply to all segments. will it affect 2.3?

Reply:
before l0, binlog/deltalog and message from mq composes the whole growing segment

Reply:
So loadStreamDelete process L0 segments only?